explainer.md

```diff
@@ -587,15 +483,50 @@ Finally, to focus an accessible node, call its focus() method.
 virtualNode.focus();
 ```
 
-When a virtual accessible node is focused, input focus in the DOM is
-unchanged. The focused accessible node is reported to assistive technology
-and other accessibility API clients, but no DOM events are fired and
-document.activeElement is unchanged.
+When a virtual accessible node is focused, 
+input focus in the DOM is unchanged. 
+The focused accessible node is reported to assistive technology
+and other accessibility API clients, 
+but no DOM events are fired and document.activeElement is unchanged.
 
 When the focused DOM element changes, accessible focus follows it:
 the DOM element's associated accessible node gets focused.
 
-### Phase 4: Full Introspection of an Accessibility Tree
+### Full Introspection of an Accessibility Tree - `ComputedAccessibleNode`
+
+```idl
+partial interface Window {
+  [NewObject] ComputedAccessibleNode getComputedAccessibleNode(Element el);
+}
+```
+
+```idl
+interface ComputedAccessibleNode {
+    // Same set of attributes as AccessibleNode, but read-only
+    readonly attribute DOMString? role;
+    readonly attribute DOMString? name;
+    
+    readonly attribute DOMString? autocomplete;
+    // ... all other ARIA-equivalent attributes
+
+    // Non-ARIA equivalent attributes
+    readonly attribute DOMString? offsetLeft;
+    readonly attribute DOMString? offsetTop;
+    readonly attribute DOMString? offsetWidth;
+    readonly attribute DOMString? offsetHeight;
+    readonly attribute AccessibleNode? offsetParent;
+    readonly boolean focusable;
+
+    readonly attribute AccessibleNode? parent;
+    readonly attribute ComputedAccessibleNode? firstChild;
+    readonly attribute ComputedAccessibleNode? lastChild;
+    readonly attribute ComputedAccessibleNode? previousSibling;
+    readonly attribute ComputedAccessibleNode? nextSibling;
+};
+
+```
+
+#### Use case 5:  Introspecting the computed tree
 
 The **Computed Accessibility Tree** API will allow authors to access
 the full computed accessibility tree -
```

Comment:
In the section below on performance you mention accessibleNode but i'm not sure if that's a leftover from a previous draft, or if you're intentionally talking about virtual nodes.